src/index.d.ts

```diff
@@ -1,7 +1,7 @@
 import http from 'http'
 import { KeyObject } from 'tls'
 import {
-    v1_1 as fspiopAPI,
+    v1_1 as fspiopAPI, // todo: use v2 of openApi
```

Comment:
Can we update this to v2?

Reply:
for now we can't just use v2 from `api-snippets`, coz there's no appropriacte export on this version:
https://github.com/mojaloop/api-snippets/blob/main/src/index.ts
So we need first to publish a new release of `api-snippets`